Libraries/TestHelpers.psm1

```diff
@@ -315,6 +315,11 @@ Function Copy-RemoteFiles($uploadTo, $downloadFrom, $downloadTo, $port, $files,
 	}
 	elseif ($download) {
 		foreach ($file in $fileList) {
+			if ($username -ne "root") {
+				$chown_cmd = "chown -Rf ${username}: $file"
+				Write-LogInfo "$chown_cmd"
+				$out = Run-LinuxCmd -username $username -password $password -ip $downloadFrom -port $port -command "$chown_cmd" -runAsSudo
+			}
 			Download-RemoteFile -downloadFrom $downloadFrom -downloadTo $downloadTo -port $port -file $file -username $username `
```

Comment:
these lines will break some platform, but we need figure out a way to fix the issue when umask value is not 022 by default.